applications/GeoMechanicsApplication/custom_elements/geo_cable_element.hpp

```diff
@@ -48,7 +48,6 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) GeoCableElement : public GeoTrussEle
     using VectorType     = Element::VectorType;
 
     using FullDofMatrixType = typename GeoTrussElementBase<TDim, TNumNodes>::FullDofMatrixType;
```

Comment:
For the GeoTrussElement, the `FullDofMatrixType` could also be removed, is it still used here?

Reply:
Yes, it is. See line 117 of file `geo_cable_element.cpp`.